### ISSUE TITLE
Error with KPoints class

### ISSUE THREAD

**Issue by matthewcarbone** [AI-multimodal/Lightshow#198]
During stress testing it turns out that it's possible for `k` to be referenced before assignment in our KPoints class:

```
self = <lightshow.common.kpoints.GenericEstimatorKpoints object at 0x172083490>
structure = Structure Summary
Lattice
    abc : 16.218902583221592 16.218902583221592 16.218902583221592
 angles : 59.999999999999...riodicSite: O (12.9, 15.77, 15.77) [0.8132, 0.5623, 0.5623]
PeriodicSite: O (12.9, 12.9, 12.9) [0.5623, 0.5623, 0.5623]

    def __call__(self, structure):
        klist = dict()
        rlatt = np.array(structure.lattice.reciprocal_lattice.abc)

        # TODO: why 10, 0.2?
        for xx in np.arange(0, 10, 0.2):
            div = np.floor(xx * rlatt) + 1
            divlatt = 2.0 * np.pi / rlatt * div

            radi = min(divlatt)

            if radi > self._max_radii:
                break
            else:
                div = tuple(div.astype(int))
                if div not in klist:
                    klist[div] = radi

        for key, value in klist.items():
            if value > self._cutoff:
                k = key
                break

>       return k
E       UnboundLocalError: local variable 'k' referenced before assignment
```

This error can be reproduced using `mp-1217206` and the default parameters. @FCMeng I think you wrote this code, right? Any ideas? 😁

**Comment by matthewcarbone** [comment]
Relatedly this is going to cause some of the checks in #197 to fail.

**Comment by matthewcarbone** [comment]
@FCMeng any thoughts on this?

**Comment by matthewcarbone** [comment]
So it appears that this happens when none of the `value`s is greater than the cutoff. And we don't have a default value for `k`. 

Intuitively, I think the default should be `k=(1, 1, 1)`. @FCMeng do you agree? I've implemented this in #197.
